index.html

```diff
@@ -17,36 +17,42 @@
             <li class="navbar-toggle">
               <img src="img/Union.png" alt="menubar" />
             </li>
+           <nav class="desktopnav">
+            <li class="portfolio" id="portfoliotxt">Portoflio</li>
+            <li class="about" id="abouttxt">About</li>
+            <li class="contactmenu" id="contactmenutxt">Contact</li>
+          </nav>
```

Comment:
- [ ] kindly consider displaying these nav links on the desktop version cause they are not visible also consider double-checking this section it has a border-left radius kindly add it and this will make your template a good duplicate of the provided design
- YOUR design 👇 
![wel](https://user-images.githubusercontent.com/60146030/221365997-7b10ec9f-f404-4c00-a316-8b385dd0e64d.PNG)
- Figma design 👇 
![wek](https://user-images.githubusercontent.com/60146030/221366066-9642386d-1393-4382-af37-0dccfedb53cc.PNG)

Reply:
Done

Reply:
done